dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5231,7 +5239,7 @@ void gp_walk(const Item* item, void* arg)
                     break;
 
                 string aliasTableName(scp->tableAlias());
-                scp->tableAlias(lower(aliasTableName));
+                scp->tableAlias(aliasTableName);
```

Comment:
This should have `lower_case_table_names` as the second argument.

Reply:
Anything put into SimpleColumn has been converted if needed. No need to spend cpu cycles doing it again.